pkg/build/controller/build/build_controller.go

```diff
@@ -926,10 +929,28 @@ func (bc *BuildController) createBuildPod(build *buildv1.Build) (*buildUpdate, e
 			return update, nil
 		}
 		glog.V(4).Infof("Recognised pod %s/%s as belonging to build %s", build.Namespace, buildPod.Name, buildDesc(build))
+		// Check if the existing pod has the CA ConfigMap properly attached
+		hasCAMap, err := bc.findBuildCAConfigMap(build, existingPod)
+		if err != nil {
+			return update, fmt.Errorf("could not find certificate authority for build: %v", err)
+		}
+		if !hasCAMap {
+			// Create the CA ConfigMap to mount certificate authorities to the existing build pod
+			update, err = bc.createBuildCAConfigMap(build, existingPod, update)
```

Comment:
not sure the point of this?  if we managed to create the build pod w/o a mounted configmap, trying to add it later is probably too late (the pod will already have started).

and of course fundamentally we should never create the build pod w/o the configmap mounted.

Reply:
> if we managed to create the build pod w/o a mounted configmap, trying to add it later is probably too late (the pod will already have started)

I'm not so sure that is the case. If the build pod is created, but the referenced ConfigMap is not created, the pod will sit in the `ContainerCreating` state.

Reply:
@bparees admittedly we are far more likely to see the pod fail to be created than the configMap.

Reply:
ok, i can see the scenario